src/library/Select/__tests__/Select.spec.js

```diff
@@ -95,6 +98,25 @@ describe('Select', () => {
     });
   });
 
+  describe('theme overrides', () => {
+    testThemeOverrides(
+      <Select data={data} id="test" isOpen>
+        <button>trigger</button>
+      </Select>,
+      getProcessedComponentThemeKeys(componentTheme, {
+        excludeKeys: [
+          'Select_color',
```

Comment:
I didn't see a reason to exclude this one.

Reply:
So, this was a fun little rabbit hole. I checked out this branch, commented out this line, and as expected, got a failing test. Upon applying `showDiffs: true` and inspecting the diff, I found `color: teal` applied to the trigger. Sure enough:

https://github.com/mineral-ui/mineral-ui/blob/master/src/library/Select/SelectTrigger.js#L139

(Which has been there since that file was added, oops: https://github.com/mineral-ui/mineral-ui/pull/581/files#diff-3568d1ea330224d13c62361ce907dc78R128)

That style is on Trigger, which is used as the "control" in FauxControl, which has its own `color` applied (https://github.com/mineral-ui/mineral-ui/blob/master/src/library/FauxControl/FauxControl.js#L140-L159). I'm pretty sure the FauxControl style clobbers the Trigger one, which is confirmed by the resulting teal color not showing up in dev tools.

But the FauxControl style references `theme.FauxControl_color` (which is mapped to `theme.TextInput_color`, which is mapped to `theme.Select_color`), so I'm not sure why updating `theme.Select_color` would have no effect.

Very likely not an issue with this PR, but it deserves further investigation, I think.

Reply:
Good catch, though I could not trace why the override was not working.  I added a FIXME to the test, removed the `color: teal` style from SelectTrigger, and will file a new issue.